subprojects/docs/src/docs/release/notes.md

```diff
@@ -63,7 +63,19 @@ java {
 }
 ```
 
-### Links to GitHub sources in Kotlin DSL Reference
+### Kotlin DSL improvements
+
+Gradle's [Kotlin DSL](userguide/kotlin_dsl.html) provides an enhanced editing experience in supported IDEs compared to the traditional Groovy DSL — auto-completion, smart content assist, quick access to documentation, navigation to source, and context-aware refactoring.
+
+Kotlin DSL has received substantial improvements in recent releases, leading to the announcement that [Kotlin DSL is Now the Default for New Gradle Builds](https://blog.gradle.org/kotlin-dsl-is-now-the-default-for-new-gradle-builds).
```

Comment:
❌ This second paragraph was contextual to the previous release and should not be included in 8.4